src/api/app/controllers/webui/request_controller.rb

```diff
@@ -155,7 +156,10 @@ def changerequest
           target.add_maintainer(@bs_request.creator) if target.can_be_modified_by?(User.possibly_nobody)
         end
       end
-      accept_request if changestate == 'accepted'
+      if changestate == 'accepted'
+        flash[:success] = "Request #{params[:number]} accepted"
+        handle_forwarding
```

Comment:
as we are setting the flash message already, shouldn't `handle_forwarding` be a job to run on background?